src/vmod_vsthrottle.vcc

```diff
@@ -95,6 +95,44 @@ Example
 		}
 
 
+$ABI vrt
```

Comment:
`ABI` should only be used once in the vcc file

Reply:
Copy/paste strikes again! Did some reading and now I understand what this is :-)

Fixed with 67f0f60